packages/medusa/src/services/cart.ts

```diff
@@ -349,7 +235,7 @@ class CartService extends TransactionBaseService {
     return await this.decorateTotals_(raw, totalsToSelect, totalsConfig)
   }
 
-  private async retrieveNew(
+  async retrieveNew(
```

Comment:
todo: we should avoid having a `[*]New` method in the public API - can we use the existing `retrieve` as a facade, not sure what we would use for splitting the logic though maybe `select` includes `subtotal | total | etc.` -> use old; otherwise use new. Wdyt?

Reply:
Yeah pretty agree with you as it introduce a two steps changes to the API which would be good to avoid. I look in your proposal and do something about it 👍